examples/acr/README.adoc

```diff
@@ -0,0 +1,34 @@
+=== Example: Azure Container Registry
```

Comment:
We are in a nasty chicken and egg situation were a sample config can't be put in `examples` folder until a release of Skaffold actually supports that config.

You should move the whole sample to `integration/examples` folder and it will be copied to top level `examples` next time a release is cut.